packages/compile/src/_shared/helpers.js

```diff
@@ -22,8 +22,17 @@ let nextLevelVarSeq = 1
 let nextAuxVarSeq = 1
 // parsed csv data cache
 let csvData = new Map()
-// string table for web apps
-export let strings = []
+
+// XXX: This is needed for tests due to sequence numbers being in module-level storage
+export function resetHelperState() {
```

Comment:
I added these reset functions for now.  Ideally we wouldn't rely on module-level (almost global-like) storage since it makes things harder to test.  But that's a bigger change that I didn't want to address here, so for now the reset functions will serve the purpose.